### PR TITLE
revert metadata

### PR DESCRIPTION
**Pull request by anjor**
Reverts https://github.com/palantir/atlasdb/pull/5467.
This isn't required as the same tracing metadata is reported via TracingCassandraClient.

### CODE REVIEW

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellValuePutter.java [hunk 1]
```diff
@@ -44,10 +44,10 @@
 
     private final LongSupplier timestampOverrideSupplier;
 
-    private CassandraKeyValueServiceConfig config;
-    private CassandraClientPool clientPool;
-    private TaskRunner taskRunner;
-    private WrappingQueryRunner queryRunner;
+    private final CassandraKeyValueServiceConfig config;
```
Comment (anjor) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellValuePutter.java:47]: this was a driveby

File: changelog/@unreleased/pr-5527.v2.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    revert metadata
+
+    Reverts https://github.com/palantir/atlasdb/pull/5467.
+    This isn't required as the same tracing metadata is reported via TracingCassandraClient.
+
+    <!---
+    Please remember to:
+    - Add any necessary release notes (including breaking changes)
+    - Make sure the documentation is up to date for your change
+    --->
```
Comment (jeremyk-91) [inline changelog/@unreleased/pr-5527.v2.yml:13]: nit: please remove the autogenerated block

**Review by jeremyk-91 — APPROVED** [review]
👍  Looks good! One small thing on the changelog
